eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/Types.hs

```diff
@@ -408,6 +420,9 @@ data LedgerState era = LedgerState
   }
   deriving (Generic)
 
+lsUTxOStateL :: Lens' (LedgerState era) (UTxOState era)
+lsUTxOStateL = lens lsUTxOState (\x y -> x {lsUTxOState = y})
```

Comment:
Same here. Defining only one lens selectively just for one field feels incomplete. If the goal is to define lenses for all of the types in this module, might as well make an extra step and at least define for all fields for the type